eos/saveddata/fit.py

```diff
@@ -1257,16 +1256,16 @@ def fits(self):
 
         return True
 
-    def __deepcopy__(self, memo):
-        copy = Fit()
+    def __deepcopy__(self, memo=None):
+        copy_ship = Fit()
         # Character and owner are not copied
-        copy.character = self.__character
-        copy.owner = self.owner
-        copy.ship = deepcopy(self.ship, memo)
```

Comment:
I agree and `deepcopy` is weird.  If you google it (which I did extensively) you find people complaining about all sorts of weird issues trying to use memos (including several versions of Python where they're broken entirely).

pyCharm clearly doesn't recognize memo as a valid parameter  (even though if you go to the function it's calling, it clearly has `Memo=None` as a parameter).

Ultimately though, we're not passing in a memo when using deepcopy.  I also validated that the object before/after the change looked the same, it copies fine.  vOv